pkg/operator/controllers/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -19,24 +19,35 @@ import (
 	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
 	mock_dynamichelper "github.com/Azure/ARO-RP/pkg/util/mocks/dynamichelper"
 	_ "github.com/Azure/ARO-RP/pkg/util/scheme"
+	utilconditions "github.com/Azure/ARO-RP/test/util/conditions"
 	utilerror "github.com/Azure/ARO-RP/test/util/error"
+	operatorv1 "github.com/openshift/api/operator/v1"
```

Comment:
Move this line up to the 2nd grouping of packages, under `ctrlfake`.

Reply:
TYSM